profile/README.md

```diff
@@ -0,0 +1,33 @@
+_Hexagonal architecture for API integrations_
+
+[![NPM](https://img.shields.io/badge/NPM-%40superfaceai-4338CA)](https://www.npmjs.com/org/superfaceai)
+[![Twitter](https://img.shields.io/badge/Twitter-%40superfaceai-4338CA)](https://twitter.com/superfaceai)
+[![Discord](https://img.shields.io/badge/Discord-superface.ai-4338CA)](https://sfc.is/discord)
```

Comment:
A few nitpicks:

- npm is officially all lowercase, it's a recursive bacronymic abbreviation for 'npm is not an acronym'
- I think our complementary purple color is `#5850ec` (cc @tjerabek to confirm)
- expanded alt

[![npm packages](https://img.shields.io/badge/npm-%40superfaceai-5850ec)](https://www.npmjs.com/org/superfaceai) [![Twitter profile](https://img.shields.io/badge/Twitter-%40superfaceai-5850ec)](https://twitter.com/superfaceai) [![Discord server](https://img.shields.io/badge/Discord-superface.ai-5850ec)](https://sfc.is/discord)

```suggestion
[![npm packages](https://img.shields.io/badge/npm-%40superfaceai-5850ec)](https://www.npmjs.com/org/superfaceai)
[![Twitter profile](https://img.shields.io/badge/Twitter-%40superfaceai-5850ec)](https://twitter.com/superfaceai)
[![Discord server](https://img.shields.io/badge/Discord-superface.ai-5850ec)](https://sfc.is/discord)
```

Reply:
@jnv color was taken from web. But Tom will come with colors.

Why the longer names?

Reply:
Accessibility and usability. If the badges fail to load for whatever reason, you want to have descriptive alternative text.